apps/web/src/components/events-display/index.tsx

```diff
@@ -96,23 +96,25 @@ async function EventList({ currentPage = 1 }: { currentPage?: number }) {
           {currentPage > 1 ? (
             <PaginationItem>
               <PaginationPrevious
-                href={`/?page=${(currentPage - 1).toFixed()}`}
+                href={`/${locale}/?page=${(currentPage - 1).toFixed()}`}
```

Comment:
`href={`?page=${(currentPage - 1).toFixed()}`}` would also work here, I think, unsure which is preferred 🤔